### PR TITLE
update style of funder logos

### PR DESCRIPTION
**Pull request by YouQam**
This PR update the style of funders logos to be aligned with corresponding project in mobile view.

### CODE REVIEW

File: src/app/public/components/about/about.component.html [hunk 1]
```diff
@@ -134,21 +140,22 @@ <h3>Klima Campus</h3>
...
 		</div>
 
-		<div>
-			<img style="width: 279px" src="../assets/logos/Logo-BMBF.png" alt="BMBF logo" />
+		<div class="supporter-desktop">
+			<img src="../assets/logos/Logo-BMBF.png" alt="BMBF logo" />
 		</div>
 
-		<div><img style="width: 442px" src="../assets/logos/Logo-BMUV.png" alt="BMUV logo" /></div>
+		<div class="supporter-desktop"><img style="width: 442px" src="../assets/logos/Logo-BMUV.png" alt="BMUV logo" /></div>
```
Comment (jona159) [inline src/app/public/components/about/about.component.html:158]: I think the width should be either part of the ```supporter-desktop```class or applied as a tailwind class

**Comment by jona159** [comment]
Mobile looks good, on desktop it looks to me like the images are not very well aligned

**Review by timber-they — APPROVED** [review]

**Comment by timber-they** [comment]
> Mobile looks good, on desktop it looks to me like the images are not very well aligned

It's not worse than on Staging though, right?

**Comment by YouQam** [comment]
> Mobile looks good, on desktop it looks to me like the images are not very well aligned

I restructured the style, now it should be aligned.

**Review by timber-they — APPROVED** [review]

**Pull request merged**